clang/cmake/caches/Peano-AIE-runtime-libraries.cmake

```diff
@@ -13,6 +13,7 @@ set(LLVM_ENABLE_RUNTIMES
       libc
     CACHE STRING "")
 set(LLVM_LIBC_FULL_BUILD ON CACHE BOOL "")
+set(LLVM_FORCE_BUILD_RUNTIME "libc" CACHE STRING "")
```

Comment:
not supported on windows technically but a nice person (@michaelrj-google) showed me the way.